src/backend/tests/Fusion.Resources.Api.Tests/IntegrationTests/DepartmentsController.cs

```diff
@@ -142,6 +142,36 @@ public async Task ShouldGetDepartmentNotInDbWhenInLineOrg()
             resp.Response.StatusCode.Should().Be(HttpStatusCode.OK);
         }
 
+        [Fact]
+        public async Task SearchShouldGetDepartmentNotInDbWhenInLineOrg()
+        {
+            var fakeResourceOwner = fixture.AddProfile(FusionAccountType.Employee);
+            var lineorgData = new
+            {
+                Count = 1,
+                TotalCount = 1,
+                Value = new[]
+                {
+                    new
+                    {
+                        fakeResourceOwner.AzureUniqueId,
+                        fakeResourceOwner.Name,
+                        fakeResourceOwner.Mail,
+                        IsResourceOwner = true,
+                        FullDepartment = "TPD LIN ORG TST"
+                    }
+                }
+            };
+
+            fixture.LineOrg.WithResponse("/lineorg/persons", lineorgData);
+
+            using var adminScope = fixture.AdminScope();
+
+            var resp = await Client.TestClientGetAsync<TestDepartment>($"/departments?$search={fakeResourceOwner.Name}&api-version=1.0-preview");
```

Comment:
Hmmm. Guess the response may be right, but you are trying to return a single item here.
Search endpoint returns a list of departments. Or?

You could even check that the response value is the same as expected ;-)

Reply:
Good catch